src/modules/crudtask.js

```diff
@@ -19,7 +19,7 @@ export function onNonEditable(elementId = 0) {
 
 export function orderArray(elementArray = []) {
   elementArray.sort((a, b) => ((a.index > b.index) ? 1 : -1));
-  for (let index = 0; index < elementArray.length; index++) { // eslint-disable-line
+  for (let index = 0; index < elementArray.length; index += 1) {
     elementArray[index].index = index;
```

Comment:
- [ ] New entries **do not match** the property index. The _property index_ must be set to the value of the _new array length_ (i.e. if a 5th task is added to the list, the index of that task should equal to 5).

**Index mismatch in your application:**
![image](https://user-images.githubusercontent.com/12745474/134726115-4e252c94-7738-4264-82b7-ebccc7febfa1.png)

Reply:
updated index in array of tasks